Civi/Api4/EckEntity.php

```diff
@@ -142,7 +142,18 @@ public static function checkAccess(string $entity_type) {
    * @return array
    */
   public static function permissions():array {
```

Comment:
This function receives `$entityName` as a param. That should help you :)

```suggestion
  public static function permissions($entityName): array {
```

Reply:
Indeed, got rid of those permission injections in the API action classes. Thanks a lot!